FlaskApp/api/app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask
+from flask_restful import Resource, Api
+from flask_jwt_extended import JWTManager
+
+from ..config import config
+from .routes import AllRides, GetRide,  JoinRequest, Register, Login, Usersg
+
+
+def create_app(config_name):
+    app = Flask(__name__)
+    app.config.from_object(config[config_name])
+    jwt = JWTManager(app)
+    api = Api(app)
+    
+    api.add_resource(Register, '/api/v1/regester')
+    api.add_resource(Login, '/api/v1/login')
+    api.add_resource(AllRides, '/api/v1/rides')
+    api.add_resource(GetRide, '/api/v1/rides/<rideId>')
+    api.add_resource( JoinRequest, '/api/v1/rides/<rideId>/requests')
+    api.add_resource(Usersg, '/api/v1/users')
```

Comment:
When the app scales, you'll have a lot of URLs. I suggest you register your URLs in a blueprint and only register the blueprints in your create_app

Reply:
Thanks ill work on that